editor/src/components/inspector/sections/component-section/data-reference-cartouche.tsx

```diff
@@ -40,12 +42,18 @@ export const DataReferenceCartoucheControl = React.memo(
       'DataReferenceCartoucheControl contentsToDisplay',
     )
 
-    // TODO get the actual data _value_ to display in the cartouche, and only show the data reference in the tooltip
+    const renderedAt =
+      props.renderedAt ??
+      // if the renderedAt is not provided, we assume that the element is rendered at the parent's children
+      EPP.create(
+        EP.parentPath(elementPath),
```

Comment:
we should turn this assumption into an explicit type, because in `getNavigatorTargets` we have all the information we need, so we can specify the behaviour there instead of guessing here

Reply:
I talked to Berci, we decided to postpone this to a future PR as we actually want to change useDataPickerButton in a way that we won't need this fallback here